{"primaryContentSections":[{"declarations":[{"tokens":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"connectError","kind":"identifier"},{"text":": ","kind":"text"},{"text":"HTTP2ErrorCode","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode","kind":"typeIdentifier","preciseIdentifier":"s:8NIOHTTP214HTTP2ErrorCodeV"}],"platforms":["Linux"],"languages":["swift"]}],"kind":"declarations"}],"sections":[],"metadata":{"fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"connectError","kind":"identifier"},{"text":": ","kind":"text"},{"text":"HTTP2ErrorCode","preciseIdentifier":"s:8NIOHTTP214HTTP2ErrorCodeV","kind":"typeIdentifier"}],"externalID":"s:8NIOHTTP214HTTP2ErrorCodeV07connectC0ACvpZ","roleHeading":"Type Property","symbolKind":"property","modules":[{"name":"NIOHTTP2"}],"title":"connectError","role":"symbol"},"kind":"symbol","variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/niohttp2\/http2errorcode\/connecterror"]}],"identifier":{"url":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode\/connectError","interfaceLanguage":"swift"},"hierarchy":{"paths":[["doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode"]]},"abstract":[{"type":"text","text":"The connection established in response to a "},{"type":"codeVoice","code":"CONNECT"},{"type":"text","text":" request"},{"type":"text","text":" "},{"type":"text","text":"was reset or abnormally closed."}],"schemaVersion":{"major":0,"minor":3,"patch":0},"references":{"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2":{"url":"\/documentation\/niohttp2","kind":"symbol","type":"topic","abstract":[],"role":"collection","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2","title":"NIOHTTP2"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/HTTP2ErrorCode/connectError":{"kind":"symbol","type":"topic","url":"\/documentation\/niohttp2\/http2errorcode\/connecterror","identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode\/connectError","fragments":[{"text":"static","kind":"keyword"},{"text":" ","kind":"text"},{"text":"let","kind":"keyword"},{"text":" ","kind":"text"},{"text":"connectError","kind":"identifier"},{"text":": ","kind":"text"},{"text":"HTTP2ErrorCode","kind":"typeIdentifier","preciseIdentifier":"s:8NIOHTTP214HTTP2ErrorCodeV"}],"role":"symbol","abstract":[{"text":"The connection established in response to a ","type":"text"},{"code":"CONNECT","type":"codeVoice"},{"text":" request","type":"text"},{"text":" ","type":"text"},{"text":"was reset or abnormally closed.","type":"text"}],"title":"connectError"},"doc://co.pointfree.VaporRouting/documentation/NIOHTTP2/HTTP2ErrorCode":{"identifier":"doc:\/\/co.pointfree.VaporRouting\/documentation\/NIOHTTP2\/HTTP2ErrorCode","abstract":[{"text":"An HTTP\/2 error code.","type":"text"}],"role":"symbol","title":"HTTP2ErrorCode","navigatorTitle":[{"kind":"identifier","text":"HTTP2ErrorCode"}],"kind":"symbol","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"HTTP2ErrorCode"}],"type":"topic","url":"\/documentation\/niohttp2\/http2errorcode"}}}